test.sh

```diff
@@ -75,6 +75,12 @@ for dir in ${VERSIONS}; do
       echo "-> Upstream tests are not present, skipping"
     fi
   fi
+
+  if [ -n "${TEST_PYTEST}" ]; then
+    VERSION=$dir python3 -m pytest -vv --showlocals test/test_*.py
```

Comment:
Well, yes, I would say more optimal solution would be to create a separate file for this. However, I would consider different naming, because this is only about OCP4 tests, isn't it?

Reply:
> solution



> Well, yes, I would say more optimal solution would be to create a separate file for this. However, I would consider different naming, because this is only about OCP4 tests, isn't it?

For now yes, but maybe later on, we will migrate also container test ;)

Reply:
Fixed by https://github.com/sclorg/container-common-scripts/pull/373/commits/a14b9f2f919e2b86e7f678e9425d4eb682fdc8c6. Good point.

Reply:
What about the separate file for test command?